src/VisualStudio/Core/Test/Progression/InheritedByGraphQueryTests.vb

```diff
@@ -74,10 +78,18 @@ using System;
 
 public interface $$I { }
 
+public class C : I { } // should appear as being derived from (implementing) I
+
+public class C2 : C { } // should not appear as being derived from (implementing ) I
+
 interface I2 : I, IComparable
 {
     void M();
 }
+
+interface I3 : I2 // should not be shown as inherited by I
```

Comment:
Can we have a VB test for these as well? VB's syntax at least treats these cases as different, I forget if the symbol model does.